content/v2.0/visualize-data/templates/view-template.md

```diff
@@ -81,6 +115,58 @@ This dashboard is designed to get you started with the Flux language. It contain
 - [`cpu` plugin](/v2.0/reference/telegraf-plugins/#cpu)
 - [`disk` plugin](/v2.0/reference/telegraf-plugins/#disk)
 
+### GitHub Data
+The GitHub Data dashboard template displays data related to GitHub repositories:
+
+- Total Forks
+- Total Size (bytes)
+- Total Stars
+- Total Open Issues
+- Forks
+- Size (bytes)
+- Stars
+- Issues
+- Summary
+
+#### Plugins
+- [`github` plugin](/v2.0/reference/telegraf-plugins/#github)
+
+
+### InfluxDB 2.0 OSS Metrics
+The InfluxDB 2.0 OSS Metrics dashboard shows a visual overview of some of the metrics
+available from the Local InfluxDB metrics endpoint located at `/metrics`.
+It displays the following information:
+
+- Uptime
+- Instance Info
+- # of Orgs
+- # of Users
+- # of Buckets
+- # of Tokens
+- # of Telegraf configurations
+- # of Dashboards
+- # of Scrapers
+- # of Tasks
+- Local Object Store IO
+- Memory Allocations (Bytes)
+- Memory Usage (%)
+- Memory Allocs & Frees (Bytes)
+
+### JMeter
+The JMeter dashboard template provides an overview of JMeter metrics.
+It displays the following:
```

Comment:
curious does it make sense to use same phrase "the following metrics:" "the following information" "the following:"